### PR TITLE
feat(Icon): Icons for app capability badges

### PR DESCRIPTION
**Pull request by cloudtx**
image

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Features
 - Add experimental runtime accessibility attributes validation (the initial step validates the Button component only) @mshoho ([#1911](https://github.com/stardust-ui/react/pull/1911))
 
+### Features
+- Export `bot`, `tab-badge` and `connector-badge` icon to Teams theme @cloudtx ([#1968](https://github.com/stardust-ui/react/pull/1968))
```
Comment (ydogandjiev) [inline CHANGELOG.md]: NIT: Why do we add `-badge` to the `tab` and `connector` icon names but not to the `bot` one? How are they different?

**Review by ydogandjiev — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cloudtx) [reply CHANGELOG.md]: I think they pulled these icons over from the angular app where they had the same naming. We could change them if need be.

**Comment by codepretty** [comment]
@TanelVari can you get filled versions of these icons?

File: CHANGELOG.md (hunk 1 above)
Comment (codepretty) [reply CHANGELOG.md]: agree - please rename to things that are generic to the image, not the use case. Like fontawesome, I think the first one should be name `robot`. The other 2 are a little more ambiguous... I'd suggest `tabs` and `plugs`

Comment (cloudtx) [reply CHANGELOG.md]: icon names changed

**Comment by cloudtx** [comment]
@codepretty Are we waiting on filled icons before moving forward?

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Export `robot`, `tabs` and `plugs` icon to Teams theme @cloudtx ([#1968](https://github.com/stardust-ui/react/pull/1968))
```
Comment (codepretty) [inline CHANGELOG.md:21]: This should go under a "### Features" heading

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/index-new.ts [hunk 3]
```diff
@@ -32,7 +32,7 @@ import processedIcons_teamcreate from './icons-team-create'
 import processedIcons_tabledelete from './icons-table-delete'
 import processedIcons_tableadd from './icons-table-add'
 import processedIcons_table from './icons-table'
-import processedIcons_tabbadge from './icons-tab-badge'
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/index-new.ts:35]: You don't need to change any of the names here

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/index.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ import processedIcons_teamcreate from './icons-team-create'
 import processedIcons_tabledelete from './icons-table-delete'
 import processedIcons_tableadd from './icons-table-add'
 import processedIcons_table from './icons-table'
-import processedIcons_tabbadge from './icons-tab-badge'
+import processedIcons_tabs from './icons-tabs'
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/index.ts:36]: You don't need to rename the icon files in `ProcessedIcons`

**Review by codepretty — APPROVED** [review]
Please update the changelog entry before checking in.

File: CHANGELOG.md (hunk 2 above)
Comment (cloudtx) [reply CHANGELOG.md:21]: updated

**Pull request closed without merging**